### PR TITLE
Updated Vagrantfile to work with the supported Mesos and Hadoop versi…

### PR DESCRIPTION
**Pull request by klucar**
…ons.

Broke provisioning into separate scripts in the ./vagrant directory.
Updated vagrant documentation
Added .npmrc that eliminates node symbolic links so build will work under Vagrant.

### CODE REVIEW

**Comment by smarella** [comment]
Thanks @klucar . Would you please add the Apache headers to the new files?

I'll review some more in the mean time.

File: Vagrantfile [hunk 1]
```diff
@@ -3,54 +3,35 @@
...
-
-apt-get -y update
-echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | sudo /usr/bin/debconf-set-selections
-apt-get -y install oracle-java8-installer
-apt-get -y install oracle-java8-set-default
-apt-get -y install libcurl3
-apt-get -y install zookeeperd
-apt-get -y install aria2
-
 MESOS_VERSION="0.21.1"
```
Comment (smarella) [inline Vagrantfile]: 0.24.1 ?

File: Vagrantfile [hunk 2]
```diff
@@ -70,11 +51,19 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.network "forwarded_port", guest: 8192, host: 8192
   config.vm.network "forwarded_port", guest: 2181, host: 2181
 
-  config.vm.provision "shell", inline: $provision_script
+  config.hostmanager.enabled = false
+  # install software
+  config.vm.provision "shell", path: "vagrant/install_java_8.sh"
```
Comment (smarella) [inline Vagrantfile]: Myriad is actually on java 7.

File: docs/vagrant.md [hunk 3]
```diff
@@ -23,51 +30,15 @@ To ssh in the cluster, run following:
...
 
-The password for vagrant user is **vagrant'**
-
-## Setting Up YARN/Hadoop ##
-
-To setup YARN/Hadoop inside VM, run following YARN setup shell files:
-
-1 Run the first YARN setup shell command from the vagrant directory to create a user hduser in group hadoop. Be sure to remember the password that you provide for this user.
-```
-cd /vagrant
```
Comment (smarella) [inline docs/vagrant.md]: Looks like the vagrant setup is a lot simpler now. Would you mind updating the wiki page as well?

https://cwiki.apache.org/confluence/display/MYRIAD/Installing+using+Vagrant

Comment (klucar) [inline docs/vagrant.md]: I'll add the headers, and switch to Java 7.

**Comment by smarella** [comment]
Ship it! :rocket:

**Pull request closed without merging**